servers/mathematica-check/pyproject.toml

```
@@ -0,0 +1,18 @@
+[project]
+name = "mathematica-check"
+version = "0.1.0"
+requires-python = ">=3.10"
+description = "A Python MCP server for interacting with Mathematica/wolframscript."
+authors = [{ name = "Your Name", email = "you@example.com" }] # Replace with your details
+license = { text = "MIT" } # Or your preferred license
+
+dependencies = [ # Specify a version range if needed
+    "mcp[cli]>=1.0.0",
+]
+
+[project.scripts]
+mathematica-check = "mathematica_server.server:main"
```

```suggestion
mcp-mathematica-check = "mathematica_server.server:main"
```


nit: Consider follow our naming convention here https://github.com/pathintegral-institute/mcp.science?tab=readme-ov-file#contributing

---

Good catch, thanks!